src/platforms/ctxlink/ctxLink_traceswo.c

```diff
@@ -0,0 +1,210 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Based on work that is Copyright (C) 2017 Black Sphere Technologies Ltd.
+ * Based on work that is Copyright (C) 2017 Dave Marples <dave@marples.net>
+ * Copyrigh (C) Sid Price 2020 <sid@sidprice.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.	 If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* This file implements capture of the TRACESWO output using ASYNC signalling.
+ *
+ * ARM DDI 0403D - ARMv7M Architecture Reference Manual
+ * ARM DDI 0337I - Cortex-M3 Technical Reference Manual
+ * ARM DDI 0314H - CoreSight Components Technical Reference Manual
+ */
+
+/* TDO/TRACESWO signal comes into the SWOUSART RX pin.
+ */
+
+#include "general.h"
+// #include "cdcacm.h"
+#include "traceswo.h"
+#include "platform.h"
+#include "WiFi_Server.h"
+
+#include <libopencm3/cm3/common.h>
+#include <libopencmsis/core_cm3.h>
+#include <libopencm3/cm3/nvic.h>
+#include <libopencm3/stm32/timer.h>
+#include <libopencm3/stm32/rcc.h>
+#include <libopencm3/stm32/usart.h>
+
+/* For speed this is set to the USB transfer size */
+#define FULL_SWO_PACKET (64)
+/* Default line rate....used as default for a request without baudrate */
+#define DEFAULTSPEED (2250000)
```

Comment:
This is lifted into swo.c in the common tree, allowing it to be dropped here. The baud rate delivered to `swo_uart_init()` (the replacement of `traceswo_init()` below) is guaranteed to be a reasonable value.

Reply:
Remove the definition.
Again note, this file is currently not being compiled into the project, and will be under major review when Wi-Fi SWO is implemented